src/components/table-component/table.stories.jsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+
+import Table from "./table";
+
+/* 
+ @author [Kurtney J.](https://github.com/xhundo)
+*/
+
+// Table Component StoryBook
+
+// Added prop control for avatar: visualization.
+export default {
+  title: "Components/Table",
+  component: Table,
+  argTypes: {
+    avatar: { control: "text" },
+    alt: { control: "text" },
+  },
+};
+
+export const Primary = (args) => (
+  <Table>
+    <Table.Header>Primary Accounts</Table.Header>
+    <Table.Cell>
+      <Table.Avatar src={args.avatar} alt={args.alt} />
+    </Table.Cell>
+  </Table>
+);
+
+Primary.args = {
+  avatar:
+    "https://cdn.pixabay.com/photo/2021/05/24/09/15/ethereum-logo-6278328_960_720.png",
+};
+
+export const Secondary = (args) => (
+  <Table>
```

Comment:
Can you add an example here with multiple sections (more than one header) to see how it looks stacked up (and update the screenshot)?

Reply:
![update](https://user-images.githubusercontent.com/98115017/216786739-46da6175-f6e9-450b-93ce-204ccde32c9a.jpg)

Reply:
That is the secondary variant with more than one header/cell, etc.